code/modules/research/xenobiology/crossbreeding/warping.dm

```diff
@@ -10,39 +10,53 @@ put up a rune with bluespace effects, lots of those runes are fluff or act as a
 	desc = "It just won't stay in place."
 	icon_state = "warping"
 	effect = "warping"
-	colour = "grey" ///default color of the crossbreed
-	var/obj/effect/warped_rune/runepath ///what runes will be drawn depending on the crossbreed color
-	var/warp_charge = 1 /// the number of "charge" a bluespace crossbreed start with
-	var/max_charge = 1 ///max number of charge, might be different depending on the crossbreed (all crossbreed have 1 max charge for now)
-	var/storing_time = 15 ///time it takes to store the rune back into the crossbreed
-	var/drawing_time = 15 ///time it takes to draw the rune
+	///default color of the crossbreed
+	colour = "grey" 
+	///what runes will be drawn depending on the crossbreed color
+	var/obj/effect/warped_rune/runepath 
+	/// the number of "charge" a bluespace crossbreed start with
+	var/warp_charge = 1 
+	///max number of charge, might be different depending on the crossbreed
+	var/max_charge = 1 
+	///time it takes to store the rune back into the crossbreed
+	var/storing_time = 15 
+	///time it takes to draw the rune
+	var/drawing_time = 15 
 
 /obj/effect/warped_rune
 	name = "warped rune"
 	desc = "An unstable rune born of the depths of bluespace"
 	icon = 'icons/obj/slimecrossing.dmi'
 	icon_state = "greyspace_rune"
-	var/runepath = /obj/effect/warped_rune  ///serves as a way for some specific runes to identify themselves in the turf.TODO: you can just use src dumbass
-	var/storing_time = 5 ///is only used for bluespace crystal erasing as of now
+	///here to avoid the rune being moved since it only sets it's turf once when it's drawn. doesn't include admin fuckery.
+	move_resist = INFINITY 
+	///serves as a way for some specific runes to identify themselves in the turf. TODO : use src instead
```

Comment:
Same here, these are comments, not variable documentation. Put them back to old format: with just `//` after the variable.